Dockerfile

```diff
@@ -15,12 +13,19 @@ RUN git config --global url."https://github.com/".insteadOf git@github.com:
 # install dependencies
 RUN npm ci
 
-COPY . /app
+# Upgrade dependencies
+FROM base AS upgrade
+RUN npm install -g npm-check-updates
```

Comment:
Hij doet toch een install van de dependencies adhv de pacages.json file? Wat doet deze upgrade dan? Gaat die dan ook de versies in de packages.json upgraden? Dat lijkt me een proces dat in de makefile moet zitten en niet in de Dockerfile. Anders dan is je Dockerfile niet, of iig een stuk minder deterministic.

Reply:
Makes sense